.travis/runcollection.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
```

Comment:
This license statement is for MIT license but AFAICS in https://github.com/linux-system-roles/network/blob/main/LICENSE the license is BSD-3. Is this intentional?

Reply:
This file is copied from https://github.com/linux-system-roles/template/blob/master/.travis/runcollection.sh where it is MIT licensed.   I suppose it doesn't matter.  Hopefully all of these copied files will go away soon . . . https://github.com/linux-system-roles/tox-lsr - so use whatever license you want

Reply:
runcollection.sh is one of the tox test script originally checked in linux-system-roles/template repo. The test scripts are under MIT. It is consistent with these files in the network repo.
```
ansible_pytest_extra_requirements.txt:# SPDX-License-Identifier: MIT
custom_requirements.txt:# SPDX-License-Identifier: MIT
molecule_extra_requirements.txt:# SPDX-License-Identifier: MIT
pylint_extra_requirements.txt:# SPDX-License-Identifier: MIT
pylintrc:# SPDX-License-Identifier: MIT
pytest_extra_requirements.txt:# SPDX-License-Identifier: MIT
tox.ini:# SPDX-License-Identifier: MIT
.travis.yml:# SPDX-License-Identifier: MIT
.yamllint_defaults.yml:# SPDX-License-Identifier: MIT
.yamllint.yml:# SPDX-License-Identifier: MIT
.travis/config.sh:# SPDX-License-Identifier: MIT
.travis/custom.sh:# SPDX-License-Identifier: MIT
.travis/runblack.sh:# SPDX-License-Identifier: MIT
.travis/runcoveralls.sh:# SPDX-License-Identifier: MIT
.travis/runflake8.sh:# SPDX-License-Identifier: MIT
.travis/runpylint.sh:# SPDX-License-Identifier: MIT
.travis/runpytest.sh:# SPDX-License-Identifier: MIT
.travis/runsyspycmd.sh:# SPDX-License-Identifier: MIT
.travis/utils.sh:# SPDX-License-Identifier: MIT
```

Reply:
Oh I didn't know where this was coming from. My bad on this. Thanks for explaining!